conanfile.py

```diff
@@ -134,30 +152,39 @@ def build(self):
                          '{nm_options}'
                          '{strip_options}'
                          '{ranlib_options}'
-                         '\''.format(vccmd=vccmd, 
-                                     prefix=prefix, 
+                         '\''.format(vccmd=vccmd,
+                                     prefix=prefix,
                                      compiler_cc=compiler_cc,
                                      compiler_cxx=compiler_cxx,
                                      ld_options=ld_options,
                                      nm_options=nm_options,
                                      strip_options=strip_options,
                                      ranlib_options=ranlib_options,
-                                     compiler_options=compiler_options, 
+                                     compiler_options=compiler_options,
                                      linker_options=linker_options,
                                      runtime=runtime_options))
 
                 self.run("{vccmd} bash -c 'make -s -j {cpus} install".format(vccmd=vccmd, cpus=tools.cpu_count()))
 
 
+    def build(self):
+        if self.settings.os != "Windows":
+            self.unix_build()
+        else:
+            self.windows_build()
+
     def package(self):
         build_src_dir = "{0}-{1}".format('ragel', self.version)
         build_dir = os.path.join('out','bin')
 
         self.copy(pattern="COPYING", dst="licenses", src=build_src_dir)
         self.copy(pattern="CREDITS", dst="licenses", src=build_src_dir)
-        self.copy(pattern="*.exe", dst="bin", src=build_dir, keep_path=False)
+        if self.settings.os != "Windows":
+            autotools = self.configure_autotools()
+            autotools.make(args=["install"])
+        else:
+            self.copy(pattern="*.exe", dst="bin", src=build_dir, keep_path=False)

```

Comment:
doesn't `make install` work on Windows?

Reply:
Good question. All Windows stuff I just moved to another method. But I'll check your suggestion.

Reply:
I've changed recipe to use install everywhere